yt/utilities/answer_testing/testing_utilities.py

```diff
@@ -0,0 +1,576 @@
+import functools
```

Comment:
This file looks very much similar to `yt/utilities/answer_testing/utils.py`. Any idea how to compare them?

Reply:
It's the same file. I just renamed it due to a comment that `utils.py` wasn't a very good file name.

Reply:
I'd suggest not comparing them. Treat it as what it is in this PR: a completely new file. This will allow to get rid of any cruft that accumulated in there over the years.